packages/core/src/config/config.ts

```diff
@@ -46,6 +46,18 @@ function getDomains() {
   return domains;
 }
 
+function getIgnoreFilePath(ignoreFile?: string, configFile?: string): string {
+  if (ignoreFile) {
+    return ignoreFile;
+  } else if (configFile) {
+    return path.join(path.dirname(configFile), IGNORE_FILE);
+  } else {
+    return process.cwd() !== 'undefined'
+      ? path.join(process.cwd(), IGNORE_FILE)
+      : ''
```

Comment:
I don't think it should return an empty string in that case. It seems more logical to me to explicitly return `undefined` if there's no file detected.